2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,18 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(families) {
+  return families.filter(familyNameChecker);
+}
+
+function familyNameChecker(familyName) {
```

Comment:
You could have used `array.split(' ')` to create an array and then check the last item in the array is "family" and check the first one begins with "A" but is not just "A"